### PR TITLE
PATAND-236: Update the autocompleteLayout Ui and some common stuff

### PR DESCRIPTION
**Pull request by jaimesDSG**
## Objective
Implement the change for the autocompleteLayout
Also include inside the modification for the submitBar, the header and the EditText. This will be use and multiple layout (if not all)

## Branches
- PAT: development
- Axon: task/MPE-1073_MPE-957
- RS: task/MPE-1073_MPE-957
- Cortex: development

#### Credentials:
- Environment: QA
- Org: hybridstudy
- Study: QA-Regression
- User: matias.battaglia+4@medable.com/qpal1010
- Task: T01 - Autocomplete text

#### Steps:
* open flask/ resumed it 
* Login if it's not the case
* Start the right task
Expected: 
* The new UI will fit the [zeplin](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5dd1fa0858eba4499fb1a32d)
* The submitBar skip button will be center
* The title on the header is remove.

Close PATAND-236

### CODE REVIEW

File: backbone/src/main/res/layout/rsb_step_layout.xml [hunk 1]
```diff
@@ -27,6 +27,13 @@
         tools:text="@string/lorem_long"
         tools:visibility="visible" />
 
+    <View
```
Comment (Gryzor) [inline backbone/src/main/res/layout/rsb_step_layout.xml:30]: Is this gray line applicable to _all_ steps who use this layout?

File: backbone/src/main/res/layout/rsb_view_submitbar.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?><!-- Change to LinearLayout for preview to work -->
 
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Gryzor) [inline backbone/src/main/res/layout/rsb_view_submitbar.xml:3]: Please don't remove this, why did you make it a LinearLayout and left it as such?

File: backbone/src/main/res/layout/rsb_view_submitbar.xml [hunk 3]
```diff
@@ -11,46 +11,46 @@
     <androidx.appcompat.widget.AppCompatTextView
         android:id="@+id/bar_submit_edit_cancel"
         style="@style/Widget.AppCompat.Button.Borderless.Colored"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:visibility="gone"
+        android:layout_weight="1"
+        android:gravity="center|left"
```
Comment (Gryzor) [inline backbone/src/main/res/layout/rsb_view_submitbar.xml:17]: left? This should be `start`. 

Also, don't use a LinearLayout + Weights.

File: backbone/src/main/res/layout/rsb_view_submitbar.xml [hunk 4]
```diff
@@ -11,46 +11,46 @@
...
-        tools:text="Submit" />
-
-    <Space
-        android:id="@+id/edit_space_view"
         android:layout_width="0dp"
-        android:visibility="gone"
         android:layout_height="wrap_content"
-        android:layout_weight="1" />
+        android:layout_weight="1"
+        android:gravity="center|right"
```
Comment (Gryzor) [inline backbone/src/main/res/layout/rsb_view_submitbar.xml:38]: no weights, and no `right`, just `end`.

File: backbone/src/main/res/layout/rsb_view_submitbar.xml [hunk 5]
```diff
@@ -11,46 +11,46 @@
...
+        tools:text="Submit" />
 
     <androidx.appcompat.widget.AppCompatTextView
         android:id="@+id/bar_submit_edit_save"
         style="@style/Widget.AppCompat.Button.Borderless.Colored"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:visibility="gone"
+        android:layout_weight="1"
```
Comment (Gryzor) [inline backbone/src/main/res/layout/rsb_view_submitbar.xml:48]: same, weight, `end`

File: backbone/src/main/res/layout/rsb_view_submitbar.xml [hunk 6]
```diff
@@ -1,56 +1,65 @@
 <?xml version="1.0" encoding="utf-8"?><!-- Change to LinearLayout for preview to work -->
 
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Gryzor) [inline backbone/src/main/res/layout/rsb_view_submitbar.xml:3]: Why remove the `<merge`? That was the important bit of the question.

Comment (jaimesDSG) [reply backbone/src/main/res/layout/rsb_view_submitbar.xml]: I fix this sorry

**Pull request merged**